From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Tue, 11 Oct 2011 09:33:29 -0600
Patch-mainline: v3.2-rc1
Subject: [PATCH] hv: hyperv.h: remove unused module macros
Git-commit: 7a4ba88cc11eb4ba72778f491d8241385cb0475d

I have no idea what these were ever for, but they aren't used, so delete
them.

Cc: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 include/linux/hyperv.h |   23 -----------------------
 1 files changed, 0 insertions(+), 23 deletions(-)

diff --git a/include/linux/hyperv.h b/include/linux/hyperv.h
index 6d9a53f..f2bf0a5 100644
--- a/include/linux/hyperv.h
+++ b/include/linux/hyperv.h
@@ -693,29 +693,6 @@ extern void vmbus_get_debug_info(struct vmbus_channel *channel,
 
 extern void vmbus_ontimer(unsigned long data);
 
-
-#define VMBUS				0x0001
-#define STORVSC				0x0002
-#define NETVSC				0x0004
-#define INPUTVSC			0x0008
-#define BLKVSC				0x0010
-#define VMBUS_DRV			0x0100
-#define STORVSC_DRV			0x0200
-#define NETVSC_DRV			0x0400
-#define INPUTVSC_DRV		0x0800
-#define BLKVSC_DRV			0x1000
-
-#define ALL_MODULES			(VMBUS		|\
-							STORVSC		|\
-							NETVSC		|\
-							INPUTVSC	|\
-							BLKVSC		|\
-							VMBUS_DRV	|\
-							STORVSC_DRV	|\
-							NETVSC_DRV	|\
-							INPUTVSC_DRV|\
-							BLKVSC_DRV)
-
 struct hv_driver;
 struct hv_device;
 
-- 
1.6.0.2


